packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -52,13 +54,7 @@ export class TeamsBotV2Impl {
   async scaffoldSourceCode(ctx: Context, inputs: Inputs): Promise<Result<Void, FxError>> {
     const workingPath = path.join(inputs.projectPath ?? "", "bot");
     const hostTypeTriggers = inputs[QuestionNames.BOT_HOST_TYPE_TRIGGER];
```

Comment:
This is a string combination of notification trigger and host, like "timer-functions", "http-restify" so it's hard to give it a better name. I think it can be improved after refactoring the question model.